studio/MetadataDetect.py

```diff
@@ -0,0 +1,18 @@
+import urllib2
```

Comment:
Should this file be committed or was it a proof of concept test ?

Reply:
Proof of concept

Reply:
proof of concept